CHANGELOG

```diff
@@ -6,6 +11,9 @@
 - Removed the deprecated (experimental) Hiera v3 backend
 - Removed the deprecated Puppet 3 function
 
+* Fri Apr 10 2020 Steven Pritchard <steven.pritchard@onyxpoint.com> - 3.1.0-0
```

Comment:
Since this is the same release, can you go ahead and roll this message into the latest commit message at the top?

Reply:
That's an old changelog message from a previous PR.  I'm not sure why/how it's being added now unless it was removed by mistake.

Reply:
Yeah, looks like a rebase/merge mistake in the last PR that was merged.